catalog/templates/base_generic.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    {% block title %}
+    <title>Local Library</title>
```

Comment:
Tìm hiểu i18n để chuyển qua đa ngôn ngữ e nhé
https://docs.djangoproject.com/en/4.2/topics/i18n/#internationalization-and-localization
Yêu cầu: Chưa cần cài đặt cho chọn ngôn ngữ, chỉ cần cài đặt để hiển thị ngôn ngữ mặc định